config/supervisor/kustomization.yaml

```diff
@@ -10,11 +10,20 @@ commonLabels:
   cluster.x-k8s.io/v1beta1: v1beta1
 resources:
   - ../base
-  - crd/vmware.infrastructure.cluster.x-k8s.io_vspheremachines.yaml
-  - crd/vmware.infrastructure.cluster.x-k8s.io_vsphereclusters.yaml
-  - crd/vmware.infrastructure.cluster.x-k8s.io_vspheremachinetemplates.yaml
-  - crd/vmware.infrastructure.cluster.x-k8s.io_vsphereclustertemplates.yaml
-  - crd/vmware.infrastructure.cluster.x-k8s.io_providerserviceaccounts.yaml
+  - crd/bases/vmware.infrastructure.cluster.x-k8s.io_vspheremachines.yaml
+  - crd/bases/vmware.infrastructure.cluster.x-k8s.io_vsphereclusters.yaml
+  - crd/bases/vmware.infrastructure.cluster.x-k8s.io_vspheremachinetemplates.yaml
+  - crd/bases/vmware.infrastructure.cluster.x-k8s.io_vsphereclustertemplates.yaml
+  - crd/bases/vmware.infrastructure.cluster.x-k8s.io_providerserviceaccounts.yaml
+  - ./webhook
+
+patches:
+  # patches here are for enabling the conversion webhook for each CRD
+  - path: crd/patches/webhook_in_vspheremachines.yaml
+
+  # patches here are for enabling the CA injection for each CRD
+  - path: crd/patches/cainjection_in_vspheremachines.yaml
```

Comment:
We don't really need this right now because we're not even having a conversion webhook. (all changes for `config/supervisor/crd` should not be needed) Maybe we should add the conversion related parts, when we introduce a new api version, and then for all CRD's :think:

Reply:
Yes, I agree, let's add them back when needed.